tox.ini

```diff
@@ -33,7 +33,7 @@ deps =
 
     perf: pyperf
 
-    tests,tests_nongpl,coverage,codecov: -r{toxinidir}/test-requirements.txt
+    tests,tests_nongpl,coverage,codecov,perf: -r{toxinidir}/test-requirements.txt
```

Comment:
Do they fail without this change?

Reply:
Yes, `tox -e perf` fails with the following error due to a missing dependency
```text
perf run-test: commands[1] | mkdir /Users/jdmoorman/Git/jsonschema/.tox/perf/tmp/benchmarks/
perf run-test: commands[2] | /Users/jdmoorman/Git/jsonschema/.tox/perf/bin/python /Users/jdmoorman/Git/jsonschema/jsonschema/benchmarks/issue232.py --inherit-environ JSON_SCHEMA_TEST_SUITE --output /Users/jdmoorman/Git/jsonschema/.tox/perf/tmp/benchmarks/issue232.json
Traceback (most recent call last):
  File "/Users/jdmoorman/Git/jsonschema/jsonschema/benchmarks/issue232.py", line 7, in <module>
    from twisted.python.filepath import FilePath
ModuleNotFoundError: No module named 'twisted'
```

Reply:
Got it.